### PR TITLE
Clarify documentation of graceful_timeout

### PR DESCRIPTION
**Pull request by abesto**

### CODE REVIEW

File: docs/source/settings.rst [hunk 1]
```diff
@@ -185,9 +185,10 @@ graceful_timeout
 
 Timeout for graceful workers restart.
 
-Generally set to thirty seconds. How max time worker can handle
-request after got restart signal. If the time is up worker will
-be force killed.
+Generally set to thirty seconds. After reciving a restart signal,
```
Comment (berkerpeksag) [inline docs/source/settings.rst]: typo: reciving -> receiving

Comment (abesto) [inline docs/source/settings.rst]: Meh. Thanks!

File: docs/source/settings.rst [hunk 2]
```diff
@@ -185,9 +185,10 @@ graceful_timeout
 
 Timeout for graceful workers restart.
 
-Generally set to thirty seconds. How max time worker can handle
-request after got restart signal. If the time is up worker will
-be force killed.
+Generally set to thirty seconds. After receiving a restart signal,
```
Comment (tilgovi) [inline docs/source/settings.rst]: I would remove this first sentence. It repeats information that is already above, in the default value.

Comment (abesto) [inline docs/source/settings.rst]: Good point!

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**